math/mathmore/inc/Math/MCParameters.h

```diff
@@ -61,7 +62,7 @@ struct VegasParameters{
    VegasParameters & operator=(const ROOT::Math::IOptions & opt);
 
    /// convert to options (return object is managed by the user)
-   ROOT::Math::IOptions * operator() () const;
+   std::unique_ptr<ROOT::Math::IOptions>  operator() () const;
```

Comment:
```suggestion
   std::unique_ptr<ROOT::Math::IOptions>  makeIOptions() const;
```
Since you are already changing the return type of that function, can we maybe also change the name?
Personally, I don't like implementing `operator()` if there is not a really good reason for it, because if is harder to `grep` the code for where it is actually called. It's better to have an explicit name I think.